tools/add_triage_signature.py

```diff
@@ -878,7 +878,11 @@ class MediaDisconnectionSignature(ErrorSignature):
     Eventually, the signature will include common i/o errors and the number of times they appeared
     '''
 
-    ERRORS_PATTERNS = 'Unable to read from the discovery media'
+    ERRORS_PATTERNS = [
+        'media was likely disconnected',
+        'Unable to read from the discovery media',
+        'Error: decoding and writing image',
```

Comment:
The third one indicates the user is faced with a cryptic coreos-installer image, it's not something we should disregard, it's something we should improve, so I don't feel it's fair to put in the same bucket as the first two

Reply:
+1 to Omer's comment; the last one is definitely not a clear user issue; it may be, but it may be not

Reply:
Hmm, should we rename the whole symptom into more generic "MediaIssues" then? Indicating that media has a problem on IO would save a lot of time during triage (even at the cost of being more vague)

Reply:
I just think we should sepearte "coreos-installer" errors from "assisted explicitly deciding we had media issues"

Reply:
OKay, reworking this to be a separate symptom